pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -82,7 +82,7 @@ let
 
   passthru = let
     # When we override the interpreter we also need to override the spliced versions of the interpreter
```

Comment:
To clarify: i meant this comment. I am not sure if it is up to date anymore.

Reply:
It is up to date and correct. The difference here is that `passthruFun` can carry derivations in the function closure too that are not filtered out.